aiida/manage/manager.py

```diff
@@ -264,6 +264,11 @@ def create_communicator(
             testing_mode=profile.is_test_profile,
         )
 
+        # Check whether a compatible version of RabbitMQ is being used.
+        self.check_rabbitmq_version(communicator)
```

Comment:
One additional comment here (but I assume that this would also affect `develop`, i.e. perhaps not for this PR):
do we know how expensive this check is and how often it is called?

We might want to add to the warning the instructions on how to disable this check via `verdi config`.

Reply:
Separate point: We might want to add to the warning the instructions on how to disable this check via `verdi config`.

Reply:
It is only called when the communicator is created, which is only once per Python interpreter typically. This was done on purpose such as to guarantee that it is displayed as soon as RabbitMQ is used (so for example just opening a shell and creating some nodes or querying won't display it) and it is shown just once.

> We might want to add to the warning the instructions on how to disable this check via verdi config.

We intentionally didn't provide this because we thought that this way people would disable it too quickly as opposed to taking the problem serious.

Reply:
The communicator is only created once, and we already decided not to publicize disabling it (it should only be done by people that know what they are doing)